manifests/plugin/remote_execution/ssh/params.pp

```diff
@@ -0,0 +1,13 @@
+# Remote Execution SSH default parameters
+class foreman_proxy::plugin::remote_execution::ssh::params {
+  include ::foreman_proxy
```

Comment:
It does for Kafo, I added it.  Puppet resolves it fine in the spec because we have the include precondition.